index.html

```diff
@@ -1,16 +1,176 @@
-    <title>Document</title>
     <link rel="stylesheet" href="style.css">
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+    </style>
+    <title>Capstone | Shubh</title>
 </head>
 <body>
-    
+    <header>
+        <!-- black header contain social icon  -->
+        <div class="black-head">
+            <ul>
+                <li><a href="www.linkedin.com/in/shubhscb"><img src="./images/Linkedin-icon.svg" width = 12 height = 12 alt=""></a></li>
+                <li><a href="#"><img src="./images/twitter-1.svg" width = 12 height = 12 alt=""></a></li>
+                <li><a href="about.html"><h5>English</h5></a></li>
+                <li><a href="#"><h5>My Page</h5></a></li>
+                <li><a href="#"><h5>logout</h5></a></li>
+            </ul>
+        </div>
+        <!-- main logo -->
+        <div class="main-header-content">
+        <div class="main-logo">
+           <a href="index.html"><img class="main-logo-body" src="./images/Orange-I-logo.jpg" width = 100 height = 70 alt=""></a> 
+        </div> 
+        <!-- header navbar -->
+        <nav class="nav-bar-desktop">
+            <ul>
+                <li class="current"><a href="index.html">Home</a></li>
+                <li><a href="about.html">About</a></li>
+                <li><a href="index.html">Program</a></li>
+                <li><a href="about.html">Contact</a></li>
+            </ul>
+        </nav>
 
+        <button class = "header-button-1" type="button">CC Campaign</button>
+
+            <!-- the hamburger -->
+            <div id="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+            <div id="pop-up-mobile-window"></div>
+        </div>
+        
+    </header>
+    <!-- showcase -->
+    <div class="showcase-mobile">
+        <div class="container-mobile">
+            <h2 class="orange-text hello-text">"Hello! Creative World"</h2>
+            <h1 class="orange-text width-half">SCHOOL OF CREATIVE <br>  WEB DESIGN <br> 2022</h1>
+            <div class="description-mobile-1">
+                <p>Lorem ipsum dolor sit amet 
+                consectetur adipisicing elit.
+                Nobis soluta sunt ex provident placeat 
+                similique eaque animi fugiat et.
+                Voluptates numquam, id dolorem at
+                esse doloremque deserunt magni excepturi!
+            </p>
+            </div>
+            <div class="announcement-mobile-1">
+                <h2 class="start-date-1">New class starts 2022.04.01 ~ (FRI)</h2>
+                <p class="location-1">@Orange Intelligence, MG Road, Bangalore</p>
+            </div>
+        </div>
+    </div>
+    <!-- lecture and curriculumn -->
+    <div class="curriculumn-mobile-grid">
+        <h2>Main Program</h2>
+        <div class="orange-line-mobile">
```

Comment:
[Optional] There is an overlap between the intro section and the `main program` section. It would be better to fix it to improve your UI design.

<img width="204" alt="image" src="https://user-images.githubusercontent.com/77190203/157849482-11132169-ab71-4de0-bf30-5aa0e52b0068.png">